src/showcase/components/input/TagInput.tsx

```diff
@@ -40,7 +49,11 @@ const TagInput: React.FC = () => {
       .map((keyword) => keyword.trim())
       .filter(Boolean)
     if (keywordsToAdd.length) {
-      const newKeywords = [...keywords, ...keywordsToAdd]
+      // const newKeywords = [...keywords, ...keywordsToAdd]
```

Comment:
If the comment is useless then let's remove it

Reply:
Check it once I removed as you said